ibis/backends/tests/test_array.py

```diff
@@ -474,6 +474,59 @@ def test_array_map(con, input, output, func):
     )
 
 
+@builtin_array
+@pytest.mark.notimpl(
+    ["datafusion", "flink", "polars", "sqlite"], raises=com.OperationNotDefinedError
+)
+@pytest.mark.notimpl(
+    ["risingwave"],
+    raises=PsycoPg2InternalError,
+    reason="TODO(Kexiang): seems a bug",
+)
+@pytest.mark.notimpl(
+    ["sqlite"], raises=com.UnsupportedBackendType, reason="Unsupported type: Array: ..."
+)
+@pytest.mark.parametrize(
+    ("input", "output"),
+    [
+        param(
+            {"a": [[1, None, 2], [4]]},
+            {"a": [[2, None, 5], [5]]},
+            marks=[
+                pytest.mark.notyet(
+                    ["bigquery"],
+                    raises=GoogleBadRequest,
+                    reason="BigQuery doesn't support arrays with null elements",
```

Comment:
In other tests we transform the result further to avoid retrieving arrays containing null values directly from the query (opting instead for maybe a count of all values or some other transformed aggregate). Not sure if that makes sense here or if it's worth it, just noting.

Reply:
The issue is in the construction of the memtable. BigQuery simply cannot hold `NULL`s elements in arrays, it's a fundamental limitation of their system.